docs/api.md

```diff
@@ -0,0 +1,5 @@
+---
+title: API
+---
+
```

Comment:
I would like to see some context setting here. 
E.g.: 
- this is the API only for Citrine-core
- the version of Citrine must match the version mentioned here. 
- base url and the tags and that modules can deploy individually
- Anything else you can think of that would be helpful in understanding :)